README.md

```diff
@@ -110,8 +110,14 @@ so it runs anywhere Git and Bash runs.
 * __Autocompletion for [Fish shell](http://fishshell.com)__
 
     ```
-    mkdir -p ~/.config/fish/completions/
-    curl -o ~/.config/fish/completions/git.fish http://raw.githubusercontent.com/Originate/git-town/master/autocomplete/git.fish
+    $ git town install-fish-autocompletion
+    ```
+
+    or manually
+
+    ```
+    $ mkdir -p ~/.config/fish/completions/
+    $ curl -o ~/.config/fish/completions/git.fish http://raw.githubusercontent.com/Originate/git-town/master/autocomplete/git.fish
```

Comment:
I see value in having all relevant installation instructions in one place in the readme. Setting Git Town up properly is the first thing people do, and at that point they are new and haven't dug through the whole documentation or man page for each command. They most likely are checking it out to see how it feels. If Git Town is not properly set up right away, and for example lacks autocompletion, it appears as a vastly inferior tool than it is, and we unnecessarily loose users. 

Telling people somewhere in a sub-page that they can also install autocompletion isn't helpful for them, nor for us. Most people don't read anything longer than a paragraph, so the installation instructions have to be concise and straight to the point.

So I'd rather keep at least the short reference to installing autocomplete in the installation instruction in the main readme, even if that is redundant. I agree it should be minimalistic, but it is at this point. 

What you propose is more logical, but less useful in real life, for real people. It isn't idiot-proof.

Does that work for you?